Dockerfile

```diff
@@ -1,19 +1,26 @@
 FROM ubuntu:14.04
 
-RUN apt-get update && \
-    apt-get -y install software-properties-common wget && \
-    apt-add-repository 'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main' && \
-    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
-    apt-get update && \
-    apt-get -y install python python-pip libpq-dev python-dev postgresql-client-9.4 && \
-    apt-get clean
+RUN apt-get update
+RUN apt-get -y install software-properties-common wget curl
+RUN apt-add-repository 'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main'
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN apt-get update
+RUN apt-get -y install python python-pip libpq-dev python-dev postgresql-client-9.4
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
+RUN apt-get install -y nodejs
+RUN apt-get clean
```

Comment:
Added individual `RUN` statements so that any future builds/changes can leverage cached layers. Let me know if you'd prefer this left as a single command and I'll change it back, Jody.